result/bgpv4.yaml

```diff
@@ -24,35 +24,89 @@ components:
           description: >-
             The list of column names that the returned result set will contain.
             If the list is empty then all columns will be returned except for
-            any result_groups.
-            The name of the BGPv4 device cannot be excluded.
+            any result_groups. The name of the BGPv4 peer cannot be excluded.
           type: array
           items:
             type: string
             enum:
               - session_state
-              - routes_advertised
-              - routes_withdrawn
+              - flap_count
+              - routes_sent
+              - routes_received
+              - route_withdraws_sent
+              - route_withdraws_received    
+              - updates_sent
+              - updates_received
+              - opens_sent
+              - opens_received
+              - keepalives_sent
+              - keepalives_received
+              - notifications_sent
+              - notifications_received
 
     Bgpv4.Metric:
       description: >-
-        BGPv4 Router statistics and learned routing information
+        BGPv4 per peer statistics information.
```

Comment:
Rephrase to: Per peer metrics for BGPv4.
This needs to be fixed for BGPv6 too.

Please remove all occurrences of `statistics` and `information`

Reply:
We feel statistics is more generic term to state about current status of a peer.